File: Java/백준/Bronze/27736. 찬반투표/찬반투표.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt();
        int sum = 0;
        int cnt = 0;
        for (int i=0;i<a;i++){
            int n = in.nextInt();
            if (n==0){
                cnt++;
            }
            sum+=n;
        }
        if (cnt*2>=a){
            System.out.println("INVALID");
        }
        else if(sum>0){
            System.out.println("APPROVED");
        }
        else{
            System.out.println("REJECTED");
        }
    }
}

File: C#/백준/Bronze/27736. 찬반투표/찬반투표.cs
using System;
using System.Linq;

namespace q{
    class w{
        static void Main(String[] args){
            int a = int.Parse(Console.ReadLine());
            int[] aa = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
            int cnt = aa.Where(n => n==0).Count();
            if (cnt*2>=a){
                Console.WriteLine("INVALID");
            }
            else if(aa.Sum()>0){
                Console.WriteLine("APPROVED");
            }
            else{
                Console.WriteLine("REJECTED");
            }
        }
    }
}